src/app/services/butterCMS.service.ts

```diff
@@ -1,4 +1,4 @@
-import * as Butter from 'buttercms';
+import Butter from 'buttercms';
```

Comment:
If this tutorial uses 1.2.2 of JS SDK, should this be `import { Butter } from 'buttercms';` per your suggestion?